### PR TITLE
feat(INJI-339): Add testID's for VC Card in home page

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**
**[INJI-339](https://mosip.atlassian.net/browse/INJI-339)**  -  Add testID, accessible, and accessibilityLabel for the VC card on the home page.

[INJI-339]: https://mosip.atlassian.net/browse/INJI-339?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: components/KebabPopUp.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = (props) => {
   const controller = useKebabPopUp(props);
   const { t } = useTranslation('HomeScreenKebabPopUp');
   return (
-    <Column>
+    <Column testID={props.testID}>
```
Comment (swatigoel) [inline components/KebabPopUp.tsx:19]: List item in overlay should have testID, that will only help in automation.
In automation, we have to click on items say pin or unpin, view activity logs so these items should have testIDs

File: components/VcItemContent.tsx [hunk 2]
```diff
@@ -118,11 +119,18 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
                 />
               )}
             </ImageBackground>
-            <Column margin="0 0 0 10">
-              {getDetails(t('fullName'), fullName, props.verifiableCredential)}
+            <Column margin="6 0 0 10">
```
Comment (swatigoel) [inline components/VcItemContent.tsx:122]: have we verified in UI, by introducing TextView UI alignment is proper?

File: components/VcItemContent.tsx [hunk 3]
```diff
@@ -118,11 +119,18 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
...
+                {getDetails(
+                  t('fullName'),
+                  fullName,
+                  props.verifiableCredential
+                )}
+              </Text>
 
               <Column margin="10 0 0 0">
                 <Text
+                  testID="idType"
```
Comment (swatigoel) [inline components/VcItemContent.tsx:133]: why nationalCard TextView doesn't have testID?

File: components/VcItemContent.tsx [hunk 4]
```diff
@@ -160,16 +168,52 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
             !props.verifiableCredential ? Theme.Styles.loadingContainer : null
           }>
           <Column>
-            {uin ? getDetails(t('uin'), uin, props.verifiableCredential) : null}
-            {vid ? getDetails(t('vid'), vid, props.verifiableCredential) : null}
+            {uin ? (
+              <Column margin="0 0 20 0">
+                <Text
+                  testID="uin"
```
Comment (swatigoel) [inline components/VcItemContent.tsx:174]: This is uin label, actual uin renderer should also have testID

File: components/VcItemContent.tsx [hunk 5]
```diff
@@ -160,16 +168,52 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
...
+                  weight="bold"
+                  size="smaller"
+                  color={Theme.Colors.DetailsLabel}>
+                  {t('uin')}
+                </Text>
+                <Text
```
Comment (swatigoel) [inline components/VcItemContent.tsx:180]: add testID to this TextView as it actually uin on UI and will be used in automation for assertion

File: components/VcItemContent.tsx [hunk 6]
```diff
@@ -160,16 +168,52 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
...
+            {vid ? (
+              <Column>
+                <Text
+                  testID="vid"
+                  weight="bold"
+                  size="smaller"
+                  color={Theme.Colors.DetailsLabel}>
+                  {t('vid')}
+                </Text>
+                <Text
```
Comment (swatigoel) [inline components/VcItemContent.tsx:198]: Same comment as for uin

File: components/VcItemContent.tsx [hunk 7]
```diff
@@ -160,16 +168,52 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
...
+                  weight="semibold"
+                  size="smaller"
+                  color={Theme.Colors.Details}>
+                  {vid}
+                </Text>
+              </Column>
+            ) : null}
             {!props.verifiableCredential
               ? getDetails(t('id'), uin || vid, props.verifiableCredential)
```
Comment (swatigoel) [inline components/VcItemContent.tsx:207]: here also add TextView to render this content and add testID

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: components/VcItemContent.tsx (hunk 2 above)
Comment (Anil-kumar-Majji) [reply components/VcItemContent.tsx:123]: Yes. It's been adjusted according to the actual one that needs to be.

File: components/VcItemContent.tsx (hunk 3 above)
Comment (Anil-kumar-Majji) [reply components/VcItemContent.tsx:151]: Now added testID's for all the values along with their respective titles.

File: components/VcItemContent.tsx (hunk 4 above)
Comment (Anil-kumar-Majji) [reply components/VcItemContent.tsx:193]: Refactored this.

File: components/VcItemContent.tsx (hunk 6 above)
Comment (Anil-kumar-Majji) [reply components/VcItemContent.tsx:218]: Done. All of them were set.

File: components/VcItemContent.tsx (hunk 7 above)
Comment (Anil-kumar-Majji) [reply components/VcItemContent.tsx:228]: Done

File: components/KebabPopUp.tsx [hunk 8]
```diff
@@ -28,7 +30,9 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = (props) => {
         overlayStyle={Theme.KebabPopUpStyles.kebabPopUp}>
         <Row style={Theme.KebabPopUpStyles.kebabHeaderStyle}>
           <View></View>
-          <Text weight="bold">{t('title')}</Text>
+          <Text testID={t('title')} weight="bold">
```
Comment (PoojaBabusing) [inline components/KebabPopUp.tsx:33]: change testID to title

**Review by swatigoel — APPROVED** [review]

**Pull request merged**